Index: vdsm/vdsm/storage/misc.py
===================================================================
--- vdsm.orig/vdsm/storage/misc.py
+++ vdsm/vdsm/storage/misc.py
@@ -187,7 +187,7 @@ def getProcCtime(pid):
     return str(ctime)
 
 _readspeed_regex = re.compile(
-    "(?P<bytes>\d+) bytes? \([\de\-.]+ [kMGT]*B\) copied, "
+    "(?P<bytes>\d+) bytes?( \([\de\-.]+ [kMGT]*B(, [\de\-.]+ [KMGTi]*B)?\))? copied, "
     "(?P<seconds>[\de\-.]+) s, "
     "([\de\-.]+|Infinity) [kMGT]*B/s"
 )
