Index: vdsm/vdsm/API.py
===================================================================
--- vdsm.orig/vdsm/API.py
+++ vdsm/vdsm/API.py
@@ -38,6 +38,7 @@ from vdsm import utils
 from clientIF import clientIF
 from vdsm import netinfo
 from vdsm import constants
+from vdsm import exception
 from vdsm import response
 import storage.misc
 import storage.clusterlock
@@ -375,7 +376,7 @@ class VM(APIBase):
         if runHooks:
             try:
                 hooks.before_get_vm_stats()
-            except hooks.HookError as e:
+            except exception.HookError as e:
                 return response.error('hookError',
                                       'Hook error: ' + str(e))
 
@@ -591,7 +592,7 @@ class VM(APIBase):
         try:
             if not v.waitForMigrationDestinationPrepare():
                 return errCode['createErr']
-        except hooks.HookError as e:
+        except exception.HookError as e:
             self.log.debug('Destination VM creation failed due to hook' +
                            ' error:' + str(e))
             return response.error('hookError', 'Destination hook failed: ' +
@@ -1495,7 +1496,7 @@ class Global(APIBase):
             if rollbackCtx['status'] != doneCode:
                 _after_network_setup_fail(networks, bondings, options)
             return rollbackCtx
-        except hooks.HookError as e:
+        except exception.HookError as e:
             _after_network_setup_fail(networks, bondings, options)
             return response.error('hookError', 'Hook error: ' + str(e))
         except:
