Index: vdsm/lib/vdsm/constants.py.in
===================================================================
--- vdsm.orig/lib/vdsm/constants.py.in
+++ vdsm/lib/vdsm/constants.py.in
@@ -77,7 +77,7 @@ P_LIBVIRT_VMCHANNELS = '/var/lib/libvirt
 P_OVIRT_VMCONSOLES = '/var/run/ovirt-vmconsole-console/'
 P_VDSM = '@VDSMDIR@/'
 P_VDSM_RPC = '@VDSMDIR@/rpc/'
-P_VDSM_HOOKS = '@HOOKSDIR@/'
+P_VDSM_HOOKS = '@CONFDIR@/hooks/'
 P_VDSM_LIB = '@VDSMLIBDIR@/'
 P_VDSM_RUN = '@VDSMRUNDIR@/'
 P_VDSM_STORAGE = P_VDSM_RUN + 'storage/'
