Index: vdsm/configure.ac
===================================================================
--- vdsm.orig/configure.ac
+++ vdsm/configure.ac
@@ -170,13 +170,13 @@ AC_SUBST([CDROMGROUP], [cdrom])
 AC_SUBST([vdsmdir], ['${datarootdir}/vdsm'])
 AC_SUBST([vdsmconfdir], ['${sysconfdir}/vdsm'])
 AC_SUBST([vdsmlogdir], ['${localstatedir}/log/vdsm'])
-AC_SUBST([vdsmrundir], ['${localstatedir}/run/vdsm'])
+AC_SUBST([vdsmrundir], ['${runstatedir}/vdsm/vdsm'])
 AC_SUBST([vdsmlibdir], ['${localstatedir}/lib/vdsm'])
 AC_SUBST([vdsmbackupdir], ['${vdsmlogdir}/backup'])
 AC_SUBST([vdsmexecdir], ['${libexecdir}/vdsm'])
 AC_SUBST([vdsmhooksdir], ['${vdsmexecdir}/hooks'])
 AC_SUBST([vdsmtsdir], ['${sysconfdir}/pki/vdsm'])
-AC_SUBST([vdsmrepo], ['/rhev/data-center'])
+AC_SUBST([vdsmrepo], ['${runstatedir}/vdsm/rhev/data-center'])
 AC_SUBST([vdsmpylibdir], ['${pythondir}/vdsm'])
 AC_SUBST([vdsminfradir], ['${vdsmpylibdir}/infra'])
 AC_SUBST([vdsmtooldir], ['${vdsmpylibdir}/tool'])
@@ -306,7 +306,7 @@ AC_PATH_PROG([REBOOT_PATH], [reboot], [/
 AC_PATH_PROG([RESTORECON_PATH], [restorecon], [/sbin/restorecon])
 AC_PATH_PROG([RM_PATH], [rm], [/bin/rm])
 AC_PATH_PROG([RSYNC_PATH], [rsync], [/usr/bin/rsync])
-AC_PATH_PROG([SAFELEASE_PATH], [safelease], [/usr/libexec/safelease/safelease])
+AC_PATH_PROG([SAFELEASE_PATH], [safelease], [/usr/bin/safelease])
 AC_PATH_PROG([SERVICE_PATH], [service], [/sbin/service])
 AC_PATH_PROG([SETSID_PATH], [setsid], [/usr/bin/setsid])
 AC_PATH_PROG([SUDO_PATH], [sudo], [/usr/bin/sudo])
Index: vdsm/init/vdsmd_init_common.sh.in
===================================================================
--- vdsm.orig/init/vdsmd_init_common.sh.in
+++ vdsm/init/vdsmd_init_common.sh.in
@@ -169,9 +169,15 @@ _DOM_METADATA_BACKUP_DIR="@VDSMLOGDIR@/b
 
 _mk_data_center() {
     local dc
-    dc="$("${_GETCONFITEM}" "${_CONF_FILE}" irs repository /rhev/)"
+    dc="$("${_GETCONFITEM}" "${_CONF_FILE}" irs repository /run/vdsm/rhev)"
     @MKDIR_P@ "${dc}"
+    @MKDIR_P@ "${dc}/data-center"
+    @MKDIR_P@ "${dc}/data-center/hsm-tasks"
+    @MKDIR_P@ "${dc}/data-center/mnt"
     "@CHOWN_PATH@" @VDSMUSER@:@VDSMGROUP@ "${dc}"
+    "@CHOWN_PATH@" @VDSMUSER@:@VDSMGROUP@ "${dc}/data-center"
+    "@CHOWN_PATH@" @VDSMUSER@:@VDSMGROUP@ "${dc}/data-center/hsm-tasks"
+    "@CHOWN_PATH@" @VDSMUSER@:@VDSMGROUP@ "${dc}/data-center/mnt"
 }
 
 _mk_dom_backup() {
@@ -191,7 +197,7 @@ _mk_core_path() {
 _mk_run_path() {
     local run_path
 
-    for run_path in "@VDSMRUNDIR@"; do
+    for run_path in "@VDSMRUNDIR@" "@VDSMRUNDIR@/payload" "@VDSMRUNDIR@/v2v" "@VDSMRUNDIR@/sourceRoutes" "@VDSMRUNDIR@/trackedInterfaces"; do
         @MKDIR_P@ "${run_path}"
         "@CHMOD_PATH@" 755 "${run_path}"
         "@CHOWN_PATH@" @VDSMUSER@:@VDSMGROUP@ "${run_path}"
