Index: vdsm/configure.ac
===================================================================
--- vdsm.orig/configure.ac
+++ vdsm/configure.ac
@@ -327,7 +327,6 @@ AC_OUTPUT([
 	Makefile
 	client/Makefile
 	contrib/Makefile
-	debian/Makefile
 	init/Makefile
 	init/systemd/Makefile
 	lib/Makefile
Index: vdsm/Makefile.am
===================================================================
--- vdsm.orig/Makefile.am
+++ vdsm/Makefile.am
@@ -21,7 +21,6 @@
 SUBDIRS = \
 	client \
 	contrib \
-	debian \
 	init \
 	lib \
 	vdsm \
